projects/whitehall/docker-compose.yml

```diff
@@ -37,7 +37,7 @@ services:
     environment:
       DATABASE_URL: "mysql2://root:root@mysql/whitehall_development"
       REDIS_URL: redis://redis
-      VIRTUAL_HOST: whitehall-admin.dev.gov.uk
+      VIRTUAL_HOST: whitehall-admin.dev.gov.uk, whitehall.dev.gov.uk
```

Comment:
I'd suggest going with whitehall-frontend.dev.gov.uk here. That would be consistent with the host whitehall is given within our system and the one that we used to use in the development VM. 

If I remember right there is also a host restriction for accessing admin so whitehall.dev.gov.uk may imply being able to access both sides but I don't think that will let you into the admin parts.

Reply:
fair point, I was going to add both whitehall and whitehall-frontend but thought it was overkill